tests/unit/ingestion/models/ingestionManager.spec.ts

```diff
@@ -60,13 +61,14 @@ describe('IngestionManager', () => {
   beforeEach(() => {
     registerDefaultConfig();
 
-    mapProxyClient = new MapProxyClient(configMock, jsLogger({ enabled: false }));
-    catalogClient = new CatalogClient(configMock, jsLogger({ enabled: false }));
-    jobManagerWrapper = new JobManagerWrapper(configMock, jsLogger({ enabled: false }));
+    mapProxyClient = new MapProxyClient(configMock, jsLogger({ enabled: false }), trace.getTracer('testTracer'));
```

Comment:
save the the 'testTracer' as const outside the 'beforeEach' scope:
```typescript
const testTracer = trace.getTracer('testTracer');
``` 

and use it for whoever needs it
```typescript
    mapProxyClient = new MapProxyClient(configMock, jsLogger({ enabled: false }), testTracer);
    catalogClient = new CatalogClient(configMock, jsLogger({ enabled: false }), testTracer);
    jobManagerWrapper = new JobManagerWrapper(configMock, jsLogger({ enabled: false }), testTracer);
```

Reply:
done